recipe_scrapers/norecipes.py

```diff
@@ -0,0 +1,61 @@
+# mypy: allow-untyped-defs
+
+from ._abstract import AbstractScraper
+from ._grouping_utils import group_ingredients
+
+
+class NoRecipes(AbstractScraper):
+    @classmethod
+    def host(cls):
+        return "norecipes.com"
+
+    def author(self):
+        return self.soup.find("meta", {"name": "author"}).get("content")
+
+    def title(self):
+        return self.schema.title()
+
+    def category(self):
+        return self.schema.category()
+
+    def total_time(self):
+        return self.schema.total_time()
+
+    def yields(self):
+        return self.schema.yields()
+
+    def image(self):
+        return self.schema.image()
+
+    def ingredients(self):
+        ingredients_list = self.schema.ingredients()
+        cleaned_ingredients = [
+            ingredient.replace("((", "(").replace("))", ")")
+            for ingredient in ingredients_list
+        ]
+        return cleaned_ingredients
+
+    def ingredient_groups(self):
+        ingredients_list = self.ingredients()
+        cleaned_ingredients = [
+            ingredient.replace("((", "(").replace("))", ")")
+            for ingredient in ingredients_list
+        ]
```

Comment:
Since the ingredient retrieval and cleanup code is the same in both of these two methods, I'd suggest extracting a private helper method to share between them (probably `def _cleaned_ingredients(self)` using the sensible naming scheme you're using already)